frontend/src/views/ContactPage.vue

```diff
@@ -466,17 +470,16 @@ export default defineComponent({
           hint="Votre email"
           type="email"
           required
-        >
-          <template #required-tip>
-            <em class="required-label"> *</em>
-          </template>
-        </DsfrInput>
+          :aria-invalid="!isMessageEmailValid"
+          aria-errormessage="email-erreur-message"
```

Comment:
Dans la doc du dsfr il y a également un attribut aria-describedby, j'en profiterais pour l'ajouterai également

```suggestion
          aria-errormessage="email-erreur-message"
          aria-describedby="email-erreur-message"
```